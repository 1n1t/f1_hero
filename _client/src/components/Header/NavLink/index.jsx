/* eslint-disable jsx-a11y/anchor-is-valid */
import React from 'react';
import { Link } from 'react-router-dom';
import PropTypes from 'prop-types';

const NavLink = ({ path, name }) => (
	<Link to={path} className="btn btn-primary">
		{name}
	</Link>
);

NavLink.propTypes = {
	path: PropTypes.string.isRequired,
	name: PropTypes.string.isRequired
};

export default NavLink;
