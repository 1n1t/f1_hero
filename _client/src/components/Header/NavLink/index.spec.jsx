import r from 'Root/jest/utils/renderer';
import NavLink from './index';

const defaultProps = {
	path: '/foo',
	name: 'Foo'
};

r.init(NavLink, defaultProps);

it('renders correctly', () => {
	const wrapper = r.render(null, true);
	expect(wrapper).toMatchSnapshot();
});
