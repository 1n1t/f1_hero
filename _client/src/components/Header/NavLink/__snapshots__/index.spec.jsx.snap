// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Link
  className="btn btn-primary"
  replace={false}
  to="/foo"
>
  Foo
</Link>
`;
